pkg/console/starter/starter.go

```diff
@@ -242,6 +242,9 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 			"Progressing",
 			"Available",
 			// "Faililng"
```

Comment:
Not for this PR, but it looks like we can remove these now?

Reply:
Yeah we should eliminate.